include/net/socket.h

```diff
@@ -144,6 +144,7 @@ int ztls_setsockopt(int sock, int level, int optname,
 #endif /* defined(CONFIG_NET_SOCKETS_SOCKOPT_TLS) */
```

Comment:
Please change the commit subject to be something like "net: sockets: offload: ...."

Reply:
OK